packages/web-components/src/avatar/avatar.spec.ts

```diff
@@ -0,0 +1,87 @@
+import { expect, test } from '@playwright/test';
+import type { Locator, Page } from '@playwright/test';
+import { fixtureURL } from '../helpers.tests.js';
+
+test.describe('Avatar Component', () => {
+  let page: Page;
+  let element: Locator;
+  let root: Locator;
+
+  test.beforeAll(async ({ browser }) => {
+    page = await browser.newPage();
+
+    element = page.locator('fluent-avatar');
+
+    root = page.locator('#root');
+
+    await page.goto(fixtureURL('components-avatar--avatar'));
+  });
+
+  test.afterAll(async () => {
+    await page.close();
+  });
+
+  test('should render without crashing', async () => {
+    await page.waitForSelector('fluent-avatar');
+  });
+
+  test('should generate initials based on the provided name value', async () => {
+    await root.evaluate(node => {
+      node.innerHTML = /* html */ `
+        <fluent-avatar name="John Doe"></fluent-avatar>
+      `;
+    });
+
+    await expect(element).toHaveText('JD');
+  });
+
+  test('should render with custom initials based on the provided initials value', async () => {
+    await root.evaluate(node => {
+      node.innerHTML = /* html */ `
+        <fluent-avatar initials="JD"></fluent-avatar>
+      `;
+    });
+
+    await expect(element).toHaveText('JD');
+  });
```

Comment:
I don't think this actually fully tests what we'd like to see. This does provide initials when initials are provided, but I think we should validate that something like a name of `Julie Wright` with custom initials provided of `JJ` would yeild, `JJ`.

Reply:
Both are probably worthwile but we should be specific, "When no name value is set" for the first, "When name value is set" for the second.